config/locales/api_particulier/cas_usages_entries.fr.yml

```
@@ -157,17 +157,17 @@ fr:
             Par ailleurs, toutes les API du bouquet API Particulier ne bénéficient pas de la modalité d'appel FranceConnect : 
 
             {:.fr-h6}
-            ### <a name="liste-api-particulier-franceconnectees"></a>Liste des API FranceConnectées
+            ### <a name="liste-api-particulier-franceconnectees"></a>Liste des API : API FranceConnectées et non FranceConnectées
```

```suggestion
            ### <a name="liste-api-particulier-franceconnectees"></a>Liste des API FranceConnectées
```

@Miryad3108 j'aurai gardé la première formulation car + percutant et + court.

---

En lisant ça, je m'attends à ne voir que des API FC du coup ça m'a assez étonné d'en voir des non FC dans la liste, d'où la proposition de modification. Mais on peut garder comme ça oui

---

On peut peut-être ajouter une petite phrase intro plutôt que de modifier le titre.